docs/papers/utils/minipaper_cracking_symmetry.py

```
@@ -0,0 +1,353 @@
+            showlegend=True
+        )
+
+        fig.show()
+
+
+from IPython.display import HTML
+from matplotlib.animation import FuncAnimation
+import numpy as np
+import matplotlib.pyplot as plt
+
+def animate_leg(theta1_vals, theta2_vals, l, a_x, a_y, b, c):
```

Ruff (0.11.9)

Missing return type annotation for public function `animate_leg`

(ANN201)